runtime.js

```diff
@@ -40,6 +40,28 @@ basisLoader.detectSupport(renderer);
 gltfLoader.setBasisLoader(basisLoader);
 basisLoader.detectSupport(renderer);
 
+const startMonetization = (instanceId, monetizationPointer, ownerAddress) => {
+  if (!monetizationPointer) {
+    return;
+  }
+
+  let monetization = document[`monetization${instanceId}`];
+  if (!monetization) {
+    document[`monetization${instanceId}`] = document.createElement('eventTarget');
```

Comment:
I don't think this is very legal; `eventTarget` is not really a DOM node name.

If we want an event target the way to do it is `new EventTarget()`.

Reply:
Though this works by chance because a DOM node called `eventTarget` is actually an `EventTarget` by inheritance, due to coincidence.

Reply:
8a77ea2